src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (firstTerm) => {
+    return (secondTerm) => {
```

Comment:
better to keep it simple in that case, I did not get what you mean by term 
```suggestion
  return (firstNumber) => {
    return (secondNumber) => {
```